homeassistant/components/assist_satellite/models.py

```diff
@@ -0,0 +1,48 @@
+"""Models for assist satellite."""
+
+from dataclasses import dataclass
+from enum import IntFlag, StrEnum
+
+
+@dataclass(frozen=True)
+class SatelliteConfig:
+    """Configuration of satellite."""
+
+    default_pipeline: str | None = None
+    """Pipeline id to use by default (None = preferred)."""
+
+    finished_speaking_seconds: float = 1.0
+    """Seconds of silence before voice command is finished."""
+
+
+class AssistSatelliteEntityFeature(IntFlag):
```

Comment:
Are any of these used ? I don't think so. Maybe we can just drop it for this PR

Reply:
Nope. I'll drop it for now.